Userland/Libraries/LibGfx/Color.cpp

```diff
@@ -43,9 +43,7 @@ String Color::to_string(HTMLCompatibleSerialization html_compatible_serializatio
     }
 
     // Otherwise, for sRGB the CSS serialization of sRGB values is used and for other color spaces, the relevant serialization of the <color> value.
-    if (alpha() < 255)
-        return MUST(String::formatted("rgba({}, {}, {}, {})", red(), green(), blue(), alpha() / 255.0));
-    return MUST(String::formatted("rgb({}, {}, {})", red(), green(), blue()));
+    return MUST(String::formatted("rgba({}, {}, {}, {})", red(), green(), blue(), alpha() / 255.0));
```

Comment:
> After checking whether alpha is equal to 255, the only other outcome is that alpha is LESS than 255. Thus no second check is required.

Not quite right. If `html_compatible_serialization == No`, we can reach here with an alpha value of 255.